src/Symfony/Component/ClassLoader/ClassCollectionLoader.php

```diff
@@ -11,10 +11,14 @@
 
 namespace Symfony\Component\ClassLoader;
 
+@trigger_error('The '.__NAMESPACE__.'\ClassCollectionLoader class is deprecated since version 3.3 and will be removed in 4.0.', E_USER_DEPRECATED);
+
 /**
  * ClassCollectionLoader.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @deprecated Deprecated since version 3.3, to be removed in 4.0.
```

Comment:
`@deprecated since version 3.3, to be removed in 4.0.` (same below: no need to duplicate the "deprecated" word)